scripts/publish.ts

```diff
@@ -167,7 +167,7 @@ async function run() {
     ? packages
     : changedFiles.reduce((changedPackages, file) => {
         const pkg = packages.find((p) =>
-          file.startsWith(path.join('packages', p.packageDir, p.srcDir)),
+          file.startsWith(path.join('packages', p.packageDir)),
```

Comment:
makes sure that we publish new versions whenever there are changes anywhere in the package directory, not just the `src` directory!